rust/ql/integration-tests/hello-project/summary.expected

```diff
@@ -15,7 +15,11 @@
 | Macro calls - resolved | 2 |
 | Macro calls - total | 2 |
 | Macro calls - unresolved | 0 |
+<<<<<<< HEAD
```

Comment:
Some merge conflict resolution gone wrong?

Reply:
Yep, I'll fix it when I get back to working on this query.

Reply:
Done.